### PR TITLE
blog post announcing community call

### PR DESCRIPTION
**Pull request by k8hertweck**
Look ok, @tracykteal?

### CODE REVIEW

File: _posts/2016/09/2016-09-09-community-call.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+layout: post
+authors: ["Tracy Teal and Kate Hertweck"]
+title: "September Carpentries Community Call"
+date: 2016-09-09
+time: "00:16:00"
+category: ["Community"]
+---
+
+Our next Carpentries Community Call (formerly called Lab Meeting or 
+Town Hall meeting) will be help Thursday, September 15. These meetings will now be
```
Comment (tracykteal) [inline _posts/2016/09/2016-09-09-community-call.md]: will be help -> will be

Comment (tracykteal) [inline _posts/2016/09/2016-09-09-community-call.md]: September 15 -> September 15 (September 16 Aus/NZ)

File: _posts/2016/09/2016-09-09-community-call.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+category: ["Community"]
+---
+
+Our next Carpentries Community Call (formerly called Lab Meeting or 
+Town Hall meeting) will be help Thursday, September 15. These meetings will now be 
+monthly on the third Thursday of every month. It would be great to 
+see instructors there! These calls are a great chance to connect with 
+other Carpentry instructors and get updates and information on 
+important and interesting topics for the community. 
+
```
Comment (tracykteal) [inline _posts/2016/09/2016-09-09-community-call.md:16]: Potentially add call times here? 

Times: 7am Pacific / 10am Eastern / 2pm UTC / 12am (Sept 16th) Sydney
             and 
            4pm Pacific / 7pm Eastern / 11pm UTC / 9am (Sept 16th) Sydney

(formatting could be different)

**Comment by tracykteal** [comment]
Added a couple comments, but looks good! OK to cross post on DC too?

File: _posts/2016/09/2016-09-09-community-call.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+authors: ["Tracy Teal and Kate Hertweck"]
```
Comment (rgaiacs) [inline _posts/2016/09/2016-09-09-community-call.md]: This is correct? I thought it should be `["Tracy Teal", "Kate Hertweck"]`.

File: _posts/2016/09/2016-09-09-community-call.md (hunk 1 above)
Comment (rgaiacs) [reply _posts/2016/09/2016-09-09-community-call.md]: Can we use September 15 HH:00 UTC+0?

**Comment by rgaiacs** [comment]
> OK to cross post on DC too?

+1

**Comment by k8hertweck** [comment]
I've made the suggested changes, should be ready to merge!

**Pull request merged**